feature-libs/cart/base/components/clear-cart/clear-cart-dialog/clear-cart-dialog.component.html

```diff
@@ -0,0 +1,65 @@
+<ng-container>
+  <div (esc)="close('Escape clicked')" class="cx-clear-cart-dialog">
+    <div class="cx-clear-cart-container">
+      <ng-container *ngIf="!(isClearing$ | async); else loading">
+        <!-- Modal Header -->
+        <div class="modal-header cx-clear-cart-header">
+          <div class="cx-clear-cart-title modal-title">
+            {{ 'clearCart.clearCart' | cxTranslate }}
+          </div>
+
+          <button
+            (click)="close('Close Clear Cart Dialog')"
```

Comment:
I don't know what is the use of the "reason" parameter, but perhaps we hand to be more precise. ( we call the dialoog close in 3 places )